### PR TITLE
Implement chest animation and sound.

### PR DESCRIPTION
**Pull request by kamcio96**
Typo, where are you? :smile:

### CODE REVIEW

File: src/main/java/net/glowstone/block/entity/TEChest.java [hunk 1]
```diff
@@ -20,4 +24,27 @@ public TEChest(GlowBlock block) {
...
+    public void addViewer() {
+        viewers++;
+        if (viewers == 1) {
+            updateInRange();
+            block.getWorld().playSound(block.getLocation(), Sound.CHEST_OPEN, 5f, 2f);
+        }
+    }
+
+    public void removeViewer() {
+        viewers--;
```
Comment (gabizou) [inline src/main/java/net/glowstone/block/entity/TEChest.java]: You should probably check that the viewers is not already 0.

Comment (kamcio96) [inline src/main/java/net/glowstone/block/entity/TEChest.java]: Player cannot close without open.
Or I don't know something.

**Comment by mastercoms** [comment]
There a few bugs if you are viewing the chest's inventory and someone else breaks the chest block for a double chest, or adds a chest block to create a double chest.

**Comment by kamcio96** [comment]
Know, I will fix this in the meantime

**Pull request merged**